app/controllers/errors_controller.rb

```diff
@@ -2,6 +2,7 @@ class ErrorsController < ApplicationController
   layout "application"
 
   def not_found
+    skip_authorization
```

Comment:
Does this need to go in a `before_action :skip_authorization, only: [:not_found, :internal_server_error, :unprocessable_entity]` ?

Reply:
Done!